modules/api/main.tf

```diff
@@ -1,5 +1,124 @@
 locals {
   app_name = "nerves_hub_api"
+  ecs_resources = var.alb ? [
+    aws_ecs_service.api_public_ecs_service[0].cluster,
+    "arn:aws:ecs:${var.region}:${var.account_id}:task-definition/nerves-hub-${terraform.workspace}-api:*",
+    ] : [
+    aws_ecs_service.api_ecs_service[0].cluster,
+    "arn:aws:ecs:${var.region}:${var.account_id}:task-definition/nerves-hub-${terraform.workspace}-api:*",
+  ]
+  ssm_prefix = "nerves_hub_api"
+
+  ecs_shared_env_vars = <<EOF
+    { "name" : "ENVIRONMENT", "value" : "${terraform.workspace}" },
+    { "name" : "APP_NAME", "value" : "${local.app_name}" },
+    { "name" : "HOST", "value" : "${aws_ssm_parameter.nerves_hub_api_ssm_host.value}" },
+    { "name" : "CLUSTER", "value" : "${aws_ssm_parameter.nerves_hub_api_ssm_cluster.value}" },
```

Comment:
I dont think these values need to be secrets, aren't they just the `host_name` from the module variable?

Reply:
Not my design - I'm just using what was here